File: Chapter-2/2/count.py
#! /usr/bin/env python3

import sys

print('What is the input string? ', end='')
sys.stdout.flush()
s = sys.stdin.readline().strip()
if len(s) == 0:
  print('Empty string!')
else:
  print('%s has %d characters.' % (s, len(s)))

File: Chapter-2/2/count.swift
#! /usr/bin/env swift
import Foundation

func main() {
  print("What is the input string? ", terminator: "")
  guard let s = readLine() else { return }
  let count = s.count
  if count == 0 {
    print("Empty string!")
  } else {
    print(String(format: "%@ has %d characters.", s, count))
  }
}
main()
